File: websocket/src/main/java/com/github/websocket/controllers/OrderController.java
package com.github.websocket.controllers;

import com.github.websocket.dto.OrderDetailDto;
import com.github.websocket.network.Broker;
import com.github.websocket.utils.Logging;
import com.netflix.hystrix.contrib.javanica.annotation.HystrixCommand;
import lombok.RequiredArgsConstructor;
import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.*;

import javax.validation.Valid;

@RestController
@RequestMapping(path = "/v1/push")
@RequiredArgsConstructor
public class OrderController {

    private final Broker broker;

    @PostMapping(path = "/edit")
    @HystrixCommand
    @ResponseStatus(HttpStatus.CREATED)
    @Logging(isTime = true, isReturn = false)
    public void pushOrder(@Valid @RequestBody OrderDetailDto payload) {
        this.broker.sendOrder(payload);
    }

}

File: server-mocks/src/main/kotlin/com/github/server/mocks/controllers/OrderController.kt
package com.github.server.mocks.controllers

import com.github.server.mocks.payload.OrderDetailEntry
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping(path = ["/v1/push"])
class OrderController {

    @PostMapping
    fun pushOrder(@RequestBody payload: OrderDetailEntry) {

    }

}